frag-nav/build.gradle

```diff
@@ -71,24 +45,38 @@ android {
                     include 'spek2'
                 }
             }
+            jacocoOptions {
```

Comment:
Do we really need all thsoe options? Also not sure about xml.enabled = false. Before default config was used?